public/app/features/dashboard-scene/settings/AnnotationsEditView.tsx

```diff
@@ -1,30 +1,107 @@
 import React from 'react';
 
-import { PageLayoutType } from '@grafana/data';
-import { SceneComponentProps, SceneObjectBase } from '@grafana/scenes';
+import { AnnotationQuery, DataTopic, PageLayoutType } from '@grafana/data';
+import {
+  SceneComponentProps,
+  SceneDataLayerProvider,
+  SceneDataLayers,
+  SceneObjectBase,
+  sceneGraph,
+} from '@grafana/scenes';
 import { Page } from 'app/core/components/Page/Page';
 
+import { DashboardScene } from '../scene/DashboardScene';
 import { NavToolbarActions } from '../scene/NavToolbarActions';
+import { dataLayersToAnnotations } from '../serialization/dataLayersToAnnotations';
 import { getDashboardSceneFor } from '../utils/utils';
 
+import { EditListViewSceneUrlSync } from './EditListViewSceneUrlSync';
+import { AnnotationSettingsList } from './annotations';
 import { DashboardEditView, DashboardEditViewState, useDashboardEditPageNav } from './utils';
 
-export interface AnnotationsEditViewState extends DashboardEditViewState {}
+export interface AnnotationsEditViewState extends DashboardEditViewState {
+  editIndex?: number | undefined;
+}
 
 export class AnnotationsEditView extends SceneObjectBase<AnnotationsEditViewState> implements DashboardEditView {
+  static Component = AnnotationsSettingsView;
+
   public getUrlKey(): string {
     return 'annotations';
   }
 
-  static Component = ({ model }: SceneComponentProps<AnnotationsEditView>) => {
-    const dashboard = getDashboardSceneFor(model);
-    const { navModel, pageNav } = useDashboardEditPageNav(dashboard, model.getUrlKey());
+  protected _urlSync = new EditListViewSceneUrlSync(this);
+
+  private get _dashboard(): DashboardScene {
+    return getDashboardSceneFor(this);
+  }
+
+  private get _dataLayers(): SceneDataLayerProvider[] {
+    return sceneGraph.getDataLayers(this._dashboard);
+  }
+
+  public getSceneDataLayers(): SceneDataLayers | undefined {
+    const data = sceneGraph.getData(this);
+
+    if (!(data instanceof SceneDataLayers)) {
+      return undefined;
+    }
+
+    return data;
+  }
```

Comment:
Any particular reason for having these two? I think the getter should be sufficient.

Reply:
You are right! Fixed in the next [PR](https://github.com/grafana/grafana/pull/81361)